index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"
+    ></script>
+    <title>Jahongir Yusupov -- Profile</title>
+  </head>
+  <body>
+    <nav class="toolbar">
+      <ul>
+        <li>
+          <p class="logo-name">Yusupov</p>
```

Comment:
- [ ] Kindly change this tag from `<p>` tag to `<a>`, this is a good practice as many sites use the logo as a link when clicked takes the user to the homepage of the website.